dashboard/app/views/levels/multi.js

```diff
@@ -163,14 +167,17 @@ Multi.prototype.ready = function () {
 
 Multi.prototype.getCurrentAnswer = function () {
   var answer;
+  var valid;
 
   if (this.numAnswers == 1) {
     answer = this.lastSelectionIndex;
+    valid = answer !== -1;
   } else {
     answer = this.selectedAnswers;
+    valid = answer !== -1;
   }
 
-  return answer;
+  return { response: answer, valid: valid };
```

Comment:
Nit: looks like you can inline var `valid` since it's the same for both branches.
```javascript
return {response: answer, valid: answer !== -1};
```